src/internal_events/loki.rs

```diff
@@ -36,17 +36,17 @@ pub struct LokiOutOfOrderEventDropped;
 
 impl InternalEvent for LokiOutOfOrderEventDropped {
     fn emit_logs(&self) {
-        warn!(
+        debug!(
             message = "Received out-of-order event; dropping event.",
             internal_log_rate_secs = 30
         );
     }
 
     fn emit_metrics(&self) {
         counter!("events_discarded_total", 1,
+                "reason" => "out_of_order"); // deprecated
+        counter!("component_discarded_events_total", 1,
                 "reason" => "out_of_order");
-        counter!("processing_errors_total", 1,
-                "error_type" => "out_of_order");
```

Comment:
We should call out in the upgrade guide that this deprecated metric was dropped.

Reply:
🤔 Do we want to just leave this deprecated this release and drop it next release?

Reply:
Yeah, that would be preferable: to go through the deprecation cycle.

Reply:
IDK why I didn't do that to begin with...